examples/document-field-customisation/keystone-server/schema.graphql

```diff
@@ -211,11 +211,11 @@ type Mutation {
 }
 
 type Query {
-  users(where: UserWhereInput! = {}, orderBy: [UserOrderByInput!]! = [], take: Int, skip: Int! = 0, cursor: UserWhereUniqueInput): [User!]
   user(where: UserWhereUniqueInput!): User
+  users(where: UserWhereInput! = {}, orderBy: [UserOrderByInput!]! = [], take: Int, skip: Int! = 0, cursor: UserWhereUniqueInput): [User!]
```

Comment:
This will be a noisy update for developers, but, this has been an inconsistency in our code for many moons, and is annoying to uphold.